source/localizable/routing/defining-your-routes.md

```diff
@@ -29,6 +29,16 @@ Router.map(function() {
 });
 ```
 
+If your route name has multiple works then your route should follow convention and be hyphenated. Routes with underscores in them will not be rendered.
```

Comment:
However, the path does allow underscores, despite convention. Should I mention this, or should we lead users to believe the route and path should both use hyphens?